snet_cli/arguments.py

```diff
@@ -249,6 +271,77 @@ def add_contract_options(parser):
         add_contract_function_options(contract_p, contract_name)
 
 
+def _add_service_publish_arguments(parser):
+    parser.add_argument("--no-register", action="store_true", help="does not register the published service")
+    parser.add_argument("--config", help="specify a custom service.json file path")
+    add_transaction_arguments(parser)
+
+
+def _add_service_update_arguments(parser):
+    parser.set_defaults(fn="update")
+    parser.add_argument("--new-price", help="new price to call the service", type=type_converter("uint256"))
+    parser.add_argument("--new-endpoint", help="new endpoint to call the service's API")
+    parser.add_argument("--new-description", help="new description for the service")
+    parser.add_argument("--config", help="specify a custom service.json file path")
+    add_transaction_arguments(parser)
+
+
+def add_service_options(parser, config):
+    parser.set_defaults(cmd=ServiceCommand)
+
+    subparsers = parser.add_subparsers(title="service commands", metavar="COMMAND")
+    subparsers.required = True
+
+    init_p = subparsers.add_parser("init", help="Initialize a service package on the filesystem")
+    init_p.set_defaults(fn="init")
+    init_p.add_argument("--name", help='name of the service to be stored in the registry (default: <current working directory>)')
+    init_p.add_argument("--model", help='local filesystem path to the service model directory (default: "model/")')
+    init_p.add_argument("--organization", help='the organization to which you want to register the service (default: "")')
+    init_p.add_argument("--price", help='initial price for interacting with the service (default: 0)')
+    init_p.add_argument("--endpoint", help="initial endpoint to call the service's API (default: \"\")")
+    init_p.add_argument("--tags", nargs="+", metavar=("TAGS", "TAG1, TAG2,"), help="tags to describe the service (default: [])")
+    init_p.add_argument("--description",
+                        help='human-readable description of the service (default: "")')
+    init_p.add_argument("-y", action="store_true", help="accept defaults for any argument that is not provided")
+
+    network_names = list(
+        map(lambda x: x[len("network."):], filter(lambda x: x.startswith("network."), config.sections())))
+
+    publish_p = subparsers.add_parser("publish", help="Publish a service to the network", default_choice="default")
+    publish_p.set_defaults(fn="publish")
+    networks_publish_subparsers = publish_p.add_subparsers(title="networks", metavar="[NETWORK]")
+
+    for network_name in network_names:
+        p = networks_publish_subparsers.add_parser(network_name, help="Publish a service to {} network".format(network_name))
+        p.set_defaults(network=network_name)
```

Comment:
nit: make this variable consistent with network command, e.g. `network_name=network_name`. Followup comment in `_get_network()` (same comment for line 320, 333, 337)